.github/workflows/protect-aem-js.yaml

```diff
@@ -0,0 +1,59 @@
+name: Block Unauthorized AEM Edits
+
+on:
+  pull_request:
```

Comment:
What if we have a PR where we are testing something and want to validate aem.js changes in boilerplate work when published? We might want to only make this rule apply to PRs against `main` or `integration`.

Reply:
Oh right - it's just a failure on the mandated checks. 👍🏼